test/functional/page_objects/unified_search_page.ts

```diff
@@ -23,4 +25,15 @@ export class UnifiedSearchPageObject extends FtrService {
     await this.browser.setLocalStorageItem('data.newDataViewMenu', 'true');
     await this.browser.refresh();
   }
+
+  public async switchDataView(switchButtonSelector: string, dataViewTitle: string) {
+    await this.testSubjects.click(switchButtonSelector);
+
+    await this.retry.waitFor(
```

Comment:
Why using a `waitFor` here? Couldn't you just increase the timeout for the same effect? Also (see other comment), I don't see how it's addressing the flakiness of the test, could you add a small explanation to your PR description?

Reply:
Right! Thanks